pyfolio/plotting.py

```diff
@@ -964,6 +965,10 @@ def plot_rolling_sharpe(returns, rolling_window=APPROX_BDAYS_PER_MONTH * 6,
     returns : pd.Series
         Daily returns of the strategy, noncumulative.
          - See full explanation in tears.create_full_tear_sheet.
+    factor_returns : pd.Series, optional
+        Daily noncumulative returns of the benchmark factor to which betas are
+        computed. Usually a benchmark such as market returns.
```

Comment:
Instead of `to which betas are computed`, here it should be `for which the benchmark rolling Sharpe is computed`, since that's what happens in this function.